core/tests/root-files-config.json

```diff
@@ -13,9 +13,7 @@
     "core/tests/test-modules-mappings/README.md",
     "core/tests/README.md",
     "assets/README.md",
-    "core/templates/domain/exploration/ExplorationObjectFactorySpec.ts",
     "extensions/rich_text_components/richTextComponentsRequires.ts",
-    "core/templates/domain/exploration/StatesObjectFactorySpec.ts",
```

Comment:
@U8NWXD, @KartikSuryavanshi has removed the following files from root-files-config.json. However, he has renamed these and not deleted them.

Even though he renamed and files and removed them completely from root-files-config.json, the test passes for checking root-files. Can there be any issues with root-files-config generator?